backend/src/openbeheer/api/tests/test_api_schema.py

```diff
@@ -83,6 +85,7 @@ class TestQueryParam(Struct, kw_only=True):
             some_string: str = field(name="some_field", default="bla")
 
         class DummyView(ListView):
+            data_type = TestQueryParam  # Echo endpoint :)
```

Comment:
Note to self: depending on which PR will be merged first, the `data_type` would need renaming: https://github.com/maykinmedia/open-beheer/pull/120

Reply:
I went here to see if this was merged or had remarks. :slightly_smiling_face: